packages/editor/src/client/global.less

```diff
@@ -0,0 +1,4 @@
+.highlighted-row .rs-table-cell-group .rs-table-cell{
+    background-color: #f2ffff;
+    transition: background-color 0.3s linear;
```

Comment:
you don't need the transition here since it's already set through the CSS from rsuite